### ISSUE TITLE
[Documentation] Updating Documentation

### ISSUE THREAD

**Issue by yoriaantje-dev** [rsjudka/intelligent-auto#16]
# Updating Documentation
Hi @rsjudka, I'd like to start helping out with your project, by mostly updating things llike your documentation and readme files and stuff. How does that sound to you?

I was thinking about adding thing along the lines off install and clone commands, and even some trouble shooting type of stuff in the Wiki perhaps? Let me know!

**Comment by rsjudka** [comment]
Hey! Yeah I think it would be great if we started a Wiki so that way we could better organize everything. I feel like the README should include just the basics while the Wiki would be more comprehensive.

**Comment by yoriaantje-dev** [comment]
I'll figure out how to make the wiki and stuff tomorrow, if I run into issues I'll let you know

**Comment by yoriaantje-dev** [comment]
I've been working on the documentation, but I haven't had much time and with all the stuff that's keep changing I've deprioritised it aswell. 
Just thought I should let you know!

**Comment by robert5974** [comment]
Hey, I forked your repo @rsjudka and have been doing some documentation updates.  I'm not done and I created a wiki which I am currently breaking up the README into seperate sections.  
If you want to check it out, keep in mind I haven't changed certain things yet (images on README, template of sidebar on Wiki, etc...).
Let me know how you feel about it so far. No rush.

**Comment by rsjudka** [comment]
@robert5974 thanks for helping out with this :)

I like the sections, especially giving the walkthrough of the UI because it isn't necessarily straightforward to know how to use it lol

My guess is you're still working out some of the markup kinks? There are some sections which would probs be cleaned up.

I actually have a logo design for the project I've kinda designed in my head, just haven't had the time to draw it up yet hah

**Comment by robert5974** [comment]
Oh yeah, I've been toying with it haha.  I just haven't fully committed to creating all the images and stuff because it changes slightly here and there with development.
If you want to explain the logo you thought of I'll whip something up.  The one I've currently chosen was minimal and I used a logo maker for a bunch of ideas and this one originally had a rabbit below the speedometer.  I removed the rabbit and made slight adjustments to see how it looked.
I've never used the markup language so I've been learning it just for this.  I haven't finished each tab but I will.  No worries, I figure it'll be helpful to me and anyone else.  Especially all the subtleties of the UI like the double tap feature :).
I thought I'd move all the instructions to the wiki page and say as much on the README.md and just list features in the README.md, but I'll make it however you'd like.

**Comment by yoriaantje-dev** [comment]
@rsjudka 
https://github.com/yoriaantje-dev/intelligent-auto/wiki

I've done the same as @rsjudka , but honestly he's done a much better job. I feel we should perhaps merge our wiki's and such. 
Also, have a look at #44, wich contains the other UDEV rules for, for example, brightness.

**Comment by yoriaantje-dev** [comment]
@robert5974
> I've done the same as @robert5974, but honestly he's done a much better job. I feel we should perhaps merge our wiki's and such.
> Also, have a look at #44, wich contains the other UDEV rules for, for example, brightness.

Maybe you can merge our wiki's, let me know

**Comment by rsjudka** [comment]
@robert5974 @yoriaantje-dev i think you both have good wikis, and i would be all for a merge of both to get as complete of a wiki as possible

one thing i noticed is that the info gets out of date really quick, so we'll need to figure out where to snap the line in terms of "this is the current blessed state" lol

**Comment by robert5974** [comment]
Agreed.  I was thinking make a pull request when you get to the point where you say this is v1.  I think I remember you mentioning that somewhere...here or on Slack.  Then we can go with more revision based changes.

Have you actually set goals for features and capabilities to be considered a v1?  Or is it really once you get all the "planned" items included?

**Comment by robert5974** [comment]
I'll start merging some of @yoriaantje-dev wiki into mine and see what differences there are.  I'll be busier starting next week as I finally go back to work and possibly starting college as well on Aug 1.  I'll find time in the evenings to iron out the documentation best I can.

**Comment by rsjudka** [comment]
I've kinda been shifting v1 around based on different requests people have :p

Right now it's looking like v1 will be when the entire reorg of aasdk/openauto/ia libraries and stuff is finished, and I think we're getting pretty close for that

**Comment by robert5974** [comment]
Ok... Awesome.  I still need to play catch up on documentation but it'll get there.

**Comment by rhysmorgan134** [comment]
@rsjudka @robert5974 Any chance of an invite to the slack? Just found the link after reading this thread, but the invite is out of date

**Comment by rsjudka** [comment]
ah sorry didnt know it expired

made a new one: https://join.slack.com/t/opendsh/shared_invite/zt-fcf38qpg-jjqFoP8v0z2tJJMYiteZHQ

this one shouldn't expire hah

**Comment by rhysmorgan134** [comment]
Awesome, thankyou 👍
